guides/session.rst

```diff
@@ -1,76 +1,114 @@
 Controlling the Browser
 =======================
 
-Ok. Now we know how to create the browser driver to talk with a specific
-browser emulator. Although we can use drivers directly to call some actions
-on the emulator, Mink provides a better way - ``Session``:
+In Mink, the entry point to the browser is called the session. Think about
+it as being your browser window (some drivers even allow to deal with switching
```

Comment:
My current opinion is that only consistency matters. I consistently use "we" in KnpU, because I think it sounds friendly :).

Here, it should be: `(some drivers even let you switch tabs!)`.